bpf/include/bpf/features_skb.h

```diff
@@ -6,14 +6,12 @@
 
 #include "features.h"
 
-/* Only skb related features. */
+/* This file is regenerated with possibly different macros at startup of the agent.
+ * See pkg/datapath/linux/probes/probes.go to add new macros.
+ */
 
-#if HAVE_PROG_TYPE_HELPER(sched_cls, bpf_skb_change_tail)
-# define BPF_HAVE_CHANGE_TAIL 1
-#endif
+/* Only skb related features. */
 
-#if HAVE_PROG_TYPE_HELPER(sched_cls, bpf_fib_lookup)
-# define BPF_HAVE_FIB_LOOKUP 1
-#endif
+#define BPF_HAVE_CHANGE_TAIL 1
```

Comment:
@borkmann We'd like to remove `BPF_HAVE_CHANGE_TAIL` from being enabled by default so this header becomes an empty stub. I see this influences the presence of `__icmp6_send_time_exceeded`, is this needed?

Reply:
Sure, but can we remove it from `features_skb.h` here? `bpf/bpf_features.h` currently contains `#define BPF__PROG_TYPE_sched_cls__HELPER_bpf_skb_change_tail 1` (this gets overwritten by bpftool at runtime), so this helper is always marked as available when building the datapath using `make -C bpf`.

If it's not important to be compiled in through the Makefile, we can remove it here to simplify.